ebuild-writing/misc-files/metadata/text.xml

```diff
@@ -280,9 +280,10 @@ There are also some attributes that can be used with these tags:
     is used, other tags with or without the restrict attribute must be
     present to cover all the versions of the package. A tag without
     the restrict attribute serves as the default. The format of the
-    restrict attribute is that of a EAPI=0 package dependency
-    specification. Due to the limitations of XML, the "&lt;" and
-    "&gt;" need to be specified using "&amp;lt;" and "&amp;gt;".
+    restrict attribute is that of a EAPI=0 package dependency specification
+    (i.e., USE-conditional or slot dependendencies are not allowed).
+    Due to the limitations of XML, the <c>&lt;</c> and <c>&gt;</c> need to
```

Comment:
Since you are already changing it, maybe replace 'limitations' with 'syntax' or sth like that. It's weird to call it a limitation if that's intended syntax.

Reply:
Right. Also, a sentence should never begin with "due to".
Fixed.